packages/client/jest.config.js

```diff
@@ -4,4 +4,9 @@
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
+  transform: {
+    "node_modules/nanoid/.+\\.(j|t)sx?$": "ts-jest",
```

Comment:
we did we need to treat `nanoid` differently and why only after the version upgrade?

Reply:
They switched to esm from commonjs between v3 and 4(current).
It means that the entire crypto lib was probably bundled inside the formers build.

Reply:
yeah but why do we need to transform it now?

Reply:
so we're overcoming a problem with Jest?

Reply:
Correct. This affects jest alone.

Reply:
I'm not saying not to use the workaround - just add a comment to `jest.config.js` explaining the *why*

Reply:
@silicakes do you have a few minutes to document this in `jest.config.js`?